src/providers/infoparser.ts

```diff
@@ -0,0 +1,165 @@
+
+
+export class REPLInfoParser {
+
+    private _name: string = undefined;
+
+    private _arglist: string = undefined;
+
+    private _formsString: string = undefined;
+
+    private _docString: string = undefined;
+
+    private _specialForm: boolean = false;
+
+    constructor(msg: any) {
+
```

Comment:
The Calva code base do not use these empty first lines for functions. Nit-picking, of course, but I see it is rather consistent in this new file, so thought it was worth a mention.

Reply:
Changed.